src/Account/AccountCollection.php

```diff
@@ -7,7 +7,7 @@
 /**
  * @author Milan Matějček
  */
-class Accounts
+class AccountCollection implements \Countable, \IteratorAggregate
```

Comment:
Me whole idea was that I have many accounts and I need to iterate them and download pays from all of them. Count is important for showing progress - i simply must know how many account I have, it's better without looking to config.

I use `IteratorAggregate` because `Iterator` is too much low-level. I was inspired by `Nette\Utils\ArrayList`.